### PR TITLE
Components creation

### PR DESCRIPTION
**Pull request by redmarverick**
- calculator dom created
- calculator design created
- removed unecessary files

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import '../styles/calculator.css';
+
+function Calculator() {
+  return (
+    <div className="calculator">
+      <div className="display">0</div>
+      <div className="keypad">
+        <button className="key" type="button">AC</button>
+        <button className="key" type="button">+/-</button>
```
Comment (abdallahmalima) [inline src/components/Calculator.js]: You have done an excellent job 💪 implementing this component. However,🙆‍♀️ based on the project requirements, it is necessary to have at least one child🙋‍♀️ component in the Calculator.js component. Incorporating a child component will enhance the modularity and structure of your code, making it easier to maintain. By separating specific functionalities into smaller, reusable components, you can improve code organization and facilitate future updates and modifications.

**Review by abdallahmalima — CHANGES_REQUESTED** [review]
Hi @redmarverick , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
- Remove  boilerplate ✔
- Components Directory✔
- `Calculator.js` file✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/keypad.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import '../styles/calculator.css';
+
+function Keypad() {
+  return (
+    <div className="keypad">
+      <button className="key" type="button">AC</button>
+      <button className="key" type="button">+/-</button>
+      <button className="key" type="button">%</button>
+      <button className="key orange" type="button">/</button>
```
Comment (TedLivist) [inline src/components/keypad.js:10]: - [ ] **[OPTIONAL]** This division sign should be &div;, not /, based on the design. You can use HTML entity code if you don't have the sign on your keyboard.

**Review by TedLivist — APPROVED** [review]
## STATUS: APPROVED ✔️ ✔️

Hi @redmarverick,

✔️ Use of functional components

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**